templates/new/template.json

```diff
@@ -6,10 +6,5 @@
             "default": "/${PROJECT_NAME}/\\w+/\\L\\0\\E/\\W+/-/",
             "pattern": "^[[:word:]-]+$"
         }
-    },
```

Comment:
See other comment. This change needs to be reverted.

Reply:
Fixed